src/representations/representation_manager_spherical_expansion.hh

```diff
@@ -261,12 +266,16 @@ namespace rascal {
         // both precomputed quantities and actual expansion coefficients
         this->radial_ortho_matrix.resize(this->max_radial, this->max_radial);
         this->fac_b.resize(this->max_radial, 1);
+        this->a_b_l_n.resize(this->max_radial, this->max_angular + 1);
```

Comment:
Odd, this sounds like a bug in hiding. But I think the line you mention is in `compute_center_contribution()`, so it's just the same factors but only for the centre (which only has a contribution for `l==0`), so I guess it's legit (though it could do with a renaming).